Tvsa (#46)
prodmgmtprog/Product-Management-Curriculum: case-studies/TvsA.md

---

## Structure

### Discovery

- Five Why's
- The Bigger Picture
- Problem Statements
- Assumptions & Validators
- User Personas
- User Research Scripting

### Definition

- Idenitfying Key Insights
- Inspiration
- Sketching
- User Journey and User Stories

### Build

- Managing a sprint
- Prioritising a backlog
- The Product Owner Role
- Sprint Reviews

---

## Reflections

- We learnt a lot from this enagagement and it was the foundation of this curriculum. Crucially, we found that within the charity sector upskilling those who will lead on digital products is an urgent need. For that purpose, we continue to deliver training programmes and maintain this curriculum.

> - Interventions should be responsive to the capacity of organisations. An organisation may already be overstretched and when this is the case, our work must adapt to their capacity. For this reason, we recommend that the core exercises are completed in workshops and additional time and support is given to those organisations who do have more time to engage.

Author: ```suggestion
- Digital interventions should be responsive to the capacity of organisations. An organisation may already be overstretched and when this is the case, our work must adapt to their capacity. For this reason, we recommend that the core exercises are completed in workshops and additional time and support is given to those organisations who do have more time to engage.
```

> - Expectations should be managed ahead of programmes beginning. For this reason, we give clear programme outlines as far in advance as reasonable and set clear expectations from the outset. For example, when asking organisations to conduct user research we'd communicate this ahead of time to give organisations appropriate time to coordinate with their users. Furthermore, we are responsive to where this may not be possible (usually due to safeguarding concerns) and adjust our outlines to accomodate this - for example, on a learning programme, conducting research with those _close to users_ is appropriate (i.e. parents for a children's charity).

Author: ```suggestion
- Expectations should be managed ahead of programmes beginning. For this reason, we give clear programme outlines as far in advance as reasonable and set clear expectations from the outset. For example, when asking organisations to conduct user research we'd communicate this ahead of time to give organisations appropriate time to coordinate with their users. Furthermore, we are responsive to where this may not be possible (usually due to safeguarding concerns) and adjust our outlines to accommodate this - for example, on a learning programme, conducting research with those _close to users_ is appropriate (i.e. parents for a children's charity). This can also cause organisations to reflect on who their services are for and who exactly to engage. 
```

> - Collaboration provides immeasurable value to charitable and social impact organisations. Embarking on a digital journey can be an intimidating experience for those unfamiliar with digital already. Idenitifying common challenges and areas of uncertainty reassures participants that they're not alone. Each charity lead benefits from working with others who have fresh perspectives, have encountered a similar challenge in the past, or have a network who'd be helpful.

Author: ```suggestion
- Collaboration provides immeasurable value to charitable and social impact organisations. Embarking on a digital journey can be an intimidating experience for those unfamiliar with digital already. Identifying common challenges and areas of uncertainty reassures participants that they're not alone. Each charity lead benefits from working with others who have fresh perspectives, have encountered a similar challenge in the past, or have a network who'd be helpful.
```
